jlm/llvm/opt/alias-analyses/PointerObjectSet.hpp

```diff
@@ -350,59 +361,171 @@ class SupersetConstraint final
   {}
 
   /**
-   * \brief Applies the constraint to the \p set
-   * \return true if this operation modified any PointerObjects or points-to-sets
+   * @return the PointerObject that should point to everything the subset points to
+   */
+  [[nodiscard]] PointerObject::Index
+  GetSuperset() const noexcept
+  {
+    return Superset_;
+  }
+
+  /**
+   * @param superset the new PointerObject that should point to everything the subset points to
+   */
+  void
+  SetSuperset(PointerObject::Index superset)
+  {
+    Superset_ = superset;
+  }
+
+  /**
+   * @return the PointerObject whose points to-set should be contained within the superset
```

Comment:
"points-to set"

Reply:
I notice I have mixed "points to-set" and "points-to-set", but I agree that in English, points-to set makes the most sense.